src/AuthorizedSocketConnection.js

```diff
@@ -150,9 +162,9 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
 
     const subscription: AsyncIterable<ExecutionResult> = (result: any);
     for await (const payload of subscription) {
-      this.log('debug', 'processing subscription update', {
-        id,
-        payload,
+      this.log('info', 'emit', {
+        messagePayload: payload,
```

Comment:
@yodaiken why is the key `messagePayload`?

Reply:
feel free to change to payload